.github/workflows/build_linux.yml

```diff
@@ -41,8 +41,8 @@ on:
         type: string
         required: false
 
-# env:
-#   CURRENT_RELEASE_BRANCH: 4.3.2
+env:
+  CURRENT_RELEASE_BRANCH: 4.3.4
```

Comment:
Typo 4.3.4/4.4.4
However this doesn't have an effect on other branches than master